modules/solid_mechanics/doc/content/source/materials/MultiPhaseStressMaterial.md

```diff
@@ -1 +1,27 @@
-!template load file=stubs/moose_object.md.template name=MultiPhaseStressMaterial syntax=/Materials/MultiPhaseStressMaterial
\ No newline at end of file
+# MultiPhaseStressMaterial
+
+!syntax description /Materials/MultiPhaseStressMaterial
```

Comment:
There is a typo in here as well. `form` should be `from`.

Reply:
fixed